labrad/backend.py

```diff
@@ -390,6 +390,7 @@ def handleResponse(self, _source, _context, request, records):
         future = self.requests[n]
         del self.requests[n]
         self.pool.add(n)
+        records = [ (ID, flat_data.unflatten()) for ID, flat_data in flat_records ]
```

Comment:
pep8: [no whitespace inside brackets](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements)